### PR TITLE
Corrects the price comments for alot of the syndicate bundles

### PR DESCRIPTION
**Pull request by 1080pCat**
## What Does This PR Do

Pretty much the title, corrects alot of the prices for the syndicate bundles, as alot of them were outdated
## Why It's Good For The Game

correct prices for referencing is good
## Testing

compiles, just comment changes

## Changelog
NPFC

### CODE REVIEW

File: code/game/objects/items/weapons/storage/uplink_kits.dm [hunk 1]
```diff
@@ -6,40 +6,40 @@
...
 		/obj/item/encryptionkey/syndicate) // 10TC
 
-/// 159
+/// 142TC
 /obj/item/storage/box/syndie_kit/bundle/agent13
 	name = "Agent 13 Bundle"
 	desc = "Find and eliminate your targets quietly and effectively with this kit."
 	items = list(
-		/obj/item/clothing/under/chameleon, // 5TC
+		/obj/item/clothing/under/chameleon, // 1TC 10tc divided over 10 items.
```
Comment (Contrabang) [inline code/game/objects/items/weapons/storage/uplink_kits.dm:33]: More readability, and standardized tc to TC
```suggestion
		/obj/item/clothing/under/chameleon, // 1TC. 10TC divided over 10 items from the chameleon kit.
```

File: code/game/objects/items/weapons/storage/uplink_kits.dm [hunk 2]
```diff
@@ -6,40 +6,40 @@
...
 		/obj/item/card/id/syndicate, // 10TC
-		/obj/item/storage/box/syndie_kit/stechkin, // 20TC
+		/obj/item/storage/box/syndie_kit/stechkin, // 26TC 20tc for stech, plus the 2 mags at 3 tc
```
Comment (Contrabang) [inline code/game/objects/items/weapons/storage/uplink_kits.dm:35]:
```suggestion
		/obj/item/storage/box/syndie_kit/stechkin, // 26TC. 20TC for stechkin, plus the 2 mags at 3TC
```

File: code/game/objects/items/weapons/storage/uplink_kits.dm [hunk 3]
```diff
@@ -57,14 +57,14 @@
...
+/// 176TC
 /obj/item/storage/box/syndie_kit/bundle/bond
 	name = "Agent 007 Bundle"
 	desc = "Shake your Martini and stir up trouble with this bundle of lethal equipment mixed with a spritz of gadgetry to keep things interesting."
 	items = list(
-		/obj/item/storage/briefcase/false_bottomed, // 20TC
+		/obj/item/storage/briefcase/false_bottomed, // 10TC
 		/obj/item/suppressor, // 5TC
-		/obj/item/storage/box/syndie_kit/stechkin, // 20TC
+		/obj/item/storage/box/syndie_kit/stechkin, // 26TC 20tc for stech, plus the 2 mags at 3 tc
```
Comment (Contrabang) [inline code/game/objects/items/weapons/storage/uplink_kits.dm:67]: ditto

File: code/game/objects/items/weapons/storage/uplink_kits.dm [hunk 4]
```diff
@@ -173,39 +173,39 @@
...
 /obj/item/storage/box/syndie_kit/bundle/grenadier
 	name = "Grenade Bundle"
 	desc = "A variety of grenades and pyrotechnics to ensure you can blast your way through any situation."
 	items = list(
-		/obj/item/storage/belt/grenade/tactical, // Contains 2 Frag and EMP grenades, 5 C4 Explosives, 5 Smoke and Gluon grenades and 1 Minibomb grenade ~20TC Estimate
-		/obj/item/storage/box/syndie_kit/stechkin, // 20TC
-		/obj/item/ammo_box/magazine/m10mm/fire, // 6TC
-		/obj/item/ammo_box/magazine/m10mm/fire, // 6TC
+		/obj/item/storage/belt/grenade/tactical, // ~60TC Contains 2 Frag and EMP grenades, 5 C4 Explosives, 5 Smoke and Gluon grenades and 1 Minibomb grenade
+		/obj/item/storage/box/syndie_kit/stechkin, // 26TC 20tc for stech, plus the 2 mags at 3 tc
```
Comment (Contrabang) [inline code/game/objects/items/weapons/storage/uplink_kits.dm:184]: ditto

File: code/game/objects/items/weapons/storage/uplink_kits.dm [hunk 5]
```diff
@@ -173,39 +173,39 @@
...
-		/obj/item/autosurgeon/organ/syndicate/oneuse/laser_arm, // ?TC
-		/obj/item/pinpointer/advpinpointer, // 20TC
-		/obj/item/autosurgeon/organ/syndicate/oneuse/hardened_heart, // Like adrenals but no speed / healing
-		/obj/item/storage/belt/utility/full/multitool, // 15TC
+		/obj/item/mod/module/stealth, // 0TC
+		/obj/item/mod/module/power_kick, // 0TC
+		/obj/item/mod/module/sphere_transform, // 0TC
+		/obj/item/autosurgeon/organ/syndicate/oneuse/laser_arm, // 0TC
+		/obj/item/pinpointer/advpinpointer, // 10TC
+		/obj/item/autosurgeon/organ/syndicate/oneuse/hardened_heart, // 0TC decent stamina regen.
```
Comment (Contrabang) [inline code/game/objects/items/weapons/storage/uplink_kits.dm:207]:
```suggestion
		/obj/item/autosurgeon/organ/syndicate/oneuse/hardened_heart, // 0TC decent stamina regen, but no speed/healing.
```

File: code/game/objects/items/weapons/storage/uplink_kits.dm [hunk 6]
```diff
@@ -173,39 +173,39 @@
...
+		/obj/item/storage/belt/utility/full/multitool, // 0TC normal toolbelt, normal tools.
```
Comment (Contrabang) [inline code/game/objects/items/weapons/storage/uplink_kits.dm:208]:
```suggestion
		/obj/item/storage/belt/utility/full/multitool, // 0TC
```

File: code/game/objects/items/weapons/storage/uplink_kits.dm [hunk 7]
```diff
@@ -214,20 +214,20 @@
...
+		/obj/item/ammo_box/a357, // 15TC
 		/obj/item/encryptionkey/syndicate, // 10 TC
-		/obj/item/clothing/under/syndicate/combat, // 0 TC
+		/obj/item/clothing/under/syndicate/combat, // 0TC
 		/obj/item/clothing/accessory/holster, // 0 TC
-		/obj/item/clothing/accessory/scarf/red, // 0 TC
+		/obj/item/clothing/accessory/scarf/red, // 0TC
 		/obj/item/clothing/head/beret, // 0 TC
 		/obj/item/clothing/gloves/combat, // 0 TC
 		/obj/item/clothing/shoes/combat) // 0 TC
```
Comment (Contrabang) [inline code/game/objects/items/weapons/storage/uplink_kits.dm:228]: All of these should be standized to `0TC` format too.

File: code/game/objects/items/weapons/storage/uplink_kits.dm [hunk 8]
```diff
@@ -237,10 +237,10 @@
 		/obj/item/encryptionkey/syndicate, // 10tc
 		/obj/item/melee/energy/sword/saber/red, // 40tc
 		/obj/item/shield/energy, // 40tc
-		/obj/item/pinpointer/advpinpointer, // 20tc, get dat fuckin disk
+		/obj/item/pinpointer/advpinpointer, // 10tc, get dat fuckin disk
```
Comment (Contrabang) [inline code/game/objects/items/weapons/storage/uplink_kits.dm:240]: While you're here, can you capitalize all these into the `10TC` format?

**Review by Contrabang — CHANGES_REQUESTED** [review]

**Review by Sirryan2002 — APPROVED** [review]

File: code/game/objects/items/weapons/storage/uplink_kits.dm [hunk 9]
```diff
@@ -214,82 +214,82 @@
...
-		/obj/item/clothing/suit/jacket/bomber/syndicate, // 3 TC
-		/obj/item/melee/knuckleduster/syndie, // 10 TC
+		/obj/item/storage/bag/plasticbag, // 1TC
+		/obj/item/grenade/clown_grenade, // 15TC
+		/obj/item/seeds/ambrosia/cruciatus, // 5TC
+		/obj/item/gun/projectile/automatic/pistol, // 20TC
+		/obj/item/ammo_box/magazine/m10mm, // 3TC
+		/obj/item/soap/syndie, // 5TC
+		/obj/item/suppressor, // 5TC
+		/obj/item/clothing/under/chameleon, // 1TC 10tc divided over 10 items.
```
Comment (DGamerL) [inline code/game/objects/items/weapons/storage/uplink_kits.dm:277]:
```suggestion
		/obj/item/clothing/under/chameleon, // 1TC. 10TC divided over 10 items.
```

**Review by DGamerL — APPROVED** [review]

**Review by Contrabang — APPROVED** [review]

**Review by Burzah — APPROVED** [review]

**Pull request merged**